Octokit/Http/HttpClientAdapter.cs

```diff
@@ -109,6 +103,8 @@ protected virtual HttpRequestMessage BuildRequestMessage(IRequest request)
             {
                 var fullUri = new Uri(request.BaseAddress, request.Endpoint);
                 requestMessage = new HttpRequestMessage(request.Method, fullUri);
+
+                requestMessage.Properties["AllowAutoRedirect"] = request.AllowAutoRedirect;
```

Comment:
@darrelmiller I'm almost inclined to obsolete `request.AllowAutoRedirect` and simplify things - as we need to control this behaviour in our very specific way. Any objections?